src/components/explorer/ExplorerIndividualContent.js

```diff
@@ -68,33 +76,71 @@ const ExplorerIndividualContent = () => {
 
     const { isFetching: individualIsFetching, data: individual } = individuals[individualID] ?? {};
 
-    // Trigger resource loading
-    useIndividualResources(individual);
+    const resources = useIndividualResources(individual);
+    useEffect(() => {
+        // Set individual resources in the store for OntologyTerm rendering with no prop drilling
+        if (resources) {
+            dispatch(setIndividualResourcesTuple(resources));
```

Comment:
if this is in the state now, maybe it should be used directly inside the ontology term to prevent all the prop-passing we have elsewhere

Reply:
That was the whole point but I forgot lol, thanks for pointing it out